packages/xo-web/src/common/selectors.js

```diff
@@ -568,3 +568,19 @@ export const getIsPoolAdmin = create(
   getCheckPermissions,
   (poolsIds, check) => some(poolsIds, poolId => check(poolId, 'administrate'))
 )
+
+export const createGetResolvedResourceSets = getResourceSets => (
+  state,
+  props
+) =>
+  createSort(
```

Comment:
You can't return a selector. A selector needs to stay the same for the whole component's life.